frontend/src/components/projects/ProjectSearch.tsx

```diff
@@ -11,21 +11,23 @@ const ProjectSearch: FunctionComponent = () => {
       <Formik
         initialValues={{ search: "" }}
         onSubmit={(values, { setSubmitting }) => {
-          fetch(`/api/v1/project_search/${values.search}`).then(
-            async (response) => {
-              if (response.status === 200) {
-                const data = await response.json();
+          if (values.search)
```

Comment:
nit: we should prefer including curly braces in cases where the if branch spans multiple lines so that the scope of the branch is clear (finding a matching `}` for the `{`)